rpc/src/lib.rs

```diff
@@ -17,6 +17,28 @@
 //! Parity RPC.
 
 #![warn(missing_docs, unused_extern_crates)]
+#![cfg_attr(feature = "cargo-clippy", warn(clippy::all, clippy::pedantic))]
+#![cfg_attr(
+	feature = "cargo-clippy",
+	allow(
+		clippy::empty_enum,
+		clippy::redundant_field_names,
```

Comment:
I think shorthand syntax has no drawbacks.

Reply:
Yes, I agree but I don't think redundant field names is bad enough to fix!

Thoughts?

Reply:
Fixed all these and remove the `ignore`